### ISSUE TITLE
Show Module level in the "Possible referenced objects" list

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#1362]
A project with three modules:

- `TwoContainerModel`
  - Spatial structure `Organism` with two containers `Container A` and `Container B`
- `Second module`
  - Spatial structure `Organism` with a container `tmp`
- `Third module`
  - Spatial structure `Organism2` with a container `test`

When defining an equation, the "Possible referenced objects" list only groups by the names of spatial structure, but not by the modules. This results in containers from the two first modules being shown under one node:

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/04e12bcd-1931-4afd-a338-27824d9ec5ce)

This might be confusing. I suggest to add an additional grouping layer by the modules.

**Comment by PavelBal** [comment]
Adding the name of the module should be sufficient.

But keep in mind, that in the above example, there should be two entries for the node "Organism", because its present in two modules.

**Comment by rwmcintosh** [comment]
Yes, understood. There will remain two spatial structures, but they are in different modules.
